providers/standard/src/airflow/providers/standard/operators/trigger_dagrun.py

```diff
@@ -264,7 +272,26 @@ def execute(self, context: Context):
             )
 
     def _trigger_dag_af_3(self, context, run_id, parsed_logical_date):
-        from airflow.providers.common.compat.sdk import DagRunTriggerException
+        try:
+            from airflow.providers.common.compat.sdk import DagRunTriggerException
+        except Exception:
+            if TYPE_CHECKING:
+                from airflow.providers.common.compat.sdk import (
```

Comment:
This method is for airflow 3, why do you think we need this catch

Reply:
I don't think this change is related to adding note to triggerdagrun operator. Can you specifically add the note and raise a different PR if you have other issues?